src/validate/validate_global_state_expression.ts

```diff
@@ -0,0 +1,22 @@
+import {getOwn} from '../util/get_own';
+import {Expression} from '../expression/expression';
+import {GlobalState} from '../expression/definitions/global_state';
+import {ValidationError} from '../error/validation_error';
+
+export function validateIfDefinedInGlobalState(key: string, options: any) {
```

Comment:
I didn't see a unit test that cover this method, did I miss it?

Reply:
Refactored end tested the exported function.